lib/classes/logger/class.FroxlorLogger.php

```diff
@@ -104,31 +104,8 @@ public function logAction ($action = USR_ACTION, $type = LOG_NOTICE, $text = nul
 			return;
 		}
 
-		if (self::$crondebug_flag) {
-			switch($type) {
-				case LOG_INFO:
-					$_type = 'information';
-					break;
-				case LOG_NOTICE:
-					$_type = 'notice';
-					break;
-				case LOG_WARNING:
-					$_type = 'warning';
-					break;
-				case LOG_ERR:
-					$_type = 'error';
-					break;
-				case LOG_CRIT:
-					$_type = 'critical';
-					break;
-				case LOG_DEBUG:
-					$_type = 'debug';
-					break;
-				default:
-					$_type = 'unknown';
-					break;
-			}
-			echo "[".$_type."] ".$text.PHP_EOL;
+		if (self::$crondebug_flag || $type <= LOG_WARNING) { // more severe levels have smaller numbers
```

Comment:
the "|| $type <= LOG_WARNING" is wrong here since we're using the logger everywhere in the panel and therefore on every warning, error, etc. it will echo the message. I'll remove that in the merge